src/app/options/options.component.spec.ts

```diff
@@ -68,4 +68,15 @@ describe('OptionsComponent', () => {
 
     expect(output.innerText).toEqual(" test1");
   });
+
+  it('should update additional options string from text in additional options textbox', async () => {
+    await fixture.whenStable();
+    const additionalOptionsTextBox = fixture.debugElement.query(By.css('.additionalOptions')).nativeElement;
```

Comment:
Please use Harnesses to test. (See notes from previous PR for guidance). https://material.angular.io/guide/using-component-harnesses#using-a-harness-to-interact-with-an-angular-material-component